### ISSUE TITLE
Files that have a comment before the @flow comment are not considered as being flow-checked

### ISSUE THREAD

**Issue by julienw** [gajus/eslint-plugin-flowtype#248]
Exemple of such a file:
```javascript
/*
  LICENSE information
*/
// @flow

var i: number = 3;
```

The problem is at https://github.com/gajus/eslint-plugin-flowtype/blob/master/src/utilities/isFlowFile.js#L17

Indeed we forcefully only handle the first comment.

I'm quite sure that flow will flow-check a file wherever the flow comment is, even if their documentation says it should appear before the first line of code. Anyway the current rule doesn't actually check this either :)

**Comment by julienw** [comment]
Fixed by #240
